addon/components/drag-sort-item.js

```diff
@@ -47,8 +48,8 @@ export default Component.extend({
     ':dragSortItem',
     'isDragged2:-isDragged',
     'isDraggingOver:-isDraggingOver',
-    'shouldShowPlaceholderAbove2:-placeholderAbove',
-    'shouldShowPlaceholderBelow2:-placeholderBelow',
+    'shouldShowPlaceholderBefore2:-placeholderAbove',
```

Comment:
You renamed the JS property, but the HTML class is still using the old name. Please update from above/below to before/after for consistency.

Reply:
Actually choose to not rename it. Didn't want to break styles for those already using the component. But, if you're fine with that, i'm happy to change the classes.

Reply:
I don't mind a breaking change, but that adds a requirement to update the readme... 😇

Reply:
I'll change it then 👍